iothub/device/samples/PnpDeviceSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -101,21 +151,77 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
                 await SendTemperatureAsync(Thermostat1, cancellationToken);
                 await SendTemperatureAsync(Thermostat2, cancellationToken);
                 await SendDeviceMemoryAsync(cancellationToken);
+                await SendDeviceHealthTelemetryAsync(cancellationToken);
 
                 temperatureReset = _temperature[Thermostat1] == 0 && _temperature[Thermostat2] == 0;
-                await Task.Delay(5 * 1000);
+                await Task.Delay(5 * 1000, CancellationToken.None);
+            }
+        }
+
+        private async Task SendInitialPropertyUpdatesAsync(string componentName, CancellationToken cancellationToken)
+        {
+            const string initialValueName = "initialValue";
+            var initialValue = new ThermostatInitialValue()
+            {
+                Temperature = 55,
+                Humidity = 68
+            };
+
+            var propertyPatch = new PropertyCollection(s_payloadConvention);
+            propertyPatch.Add(initialValueName, initialValue, componentName);
+
+            await _deviceClient.UpdatePropertiesAsync(propertyPatch, cancellationToken);
```

Comment:
verify that service doesn't actually return anything as a result of this operation